project/plugins.sbt

```diff
@@ -10,8 +10,6 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.5.0")
```

Comment:
Extracting structure fails without sbt-artifactory plugin... please add this

Reply:
There was a message posted in Slack, that referenced the Confluence page:https://confluence.tools.tax.service.gov.uk/display/TEC/2021/01/18/Removing+the+sbt-artifactory+plugin

Reply:
I have no problems running the project locally with the plug in removed.